frontend/src/employee-frontend/components/child-information/Placements.tsx

```diff
@@ -60,7 +60,8 @@ export default React.memo(function Placements({ id, startOpen }: Props) {
               'SERVICE_WORKER',
               'UNIT_SUPERVISOR',
               'FINANCE_ADMIN',
-              'ADMIN'
+              'ADMIN',
+              'EARLY_CHILDHOOD_EDUCATION_SECRETARY'
```

Comment:
It should be possible to replace this with a check against `permittedActions` from `unitQuery`, but perhaps it's not relevant for this PR.